src/main/java/org/broadinstitute/consent/http/service/VoteService.java

```diff
@@ -289,30 +283,25 @@ public void sendDatasetApprovalNotifications(List<Vote> votes, User user) {
       logException("Error indexing datasets for approved DARs: " + e.getMessage(), e);
     }
 
-    // For each dar collection, email the researcher summarizing the approved datasets in that collection
-    collections.forEach(c -> {
+    // For each dar, email the researcher summarizing the approved datasets in that dar
```

Comment:
Work on DARs, not collection objects because we don't want to take all of the actions contemplated by the original code on all of the DARs in the collection.  We only want to work on the DARs from the elections that were tied to the votes provided as input to this method.